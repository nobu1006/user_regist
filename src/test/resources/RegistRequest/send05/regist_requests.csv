id,email,unique_key,regist_date,del_flg
