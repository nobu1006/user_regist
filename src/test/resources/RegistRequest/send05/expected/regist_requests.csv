email,unique_key,del_flg
nobuhiko.tobita@gmail.com,5af48f3b-468b-4ae0-a065-7d7ac70b37a8,false
